### PR TITLE
Update index.md file

### PR DESCRIPTION
**Pull request by mrmakadia94**
translation start in index.md file

### CODE REVIEW

**Comment by arshadkazmi42** [comment]
@mrmakadia94 You have not translated the complete file. Can you translate the complete file and then re-request for review.
I would be happy to review this once the translation is complete. 
Thanks

**Comment by mrmakadia94** [comment]
Hello @arshadkazmi42 
Thanks to connect
i will give you update once i completed the file.

**Comment by mrmakadia94** [comment]
Hello @arshadkazmi42 
please check the pull request i just completed the translation of file.

Thanks

File: docs/docs/index.md [hunk 1]
```diff
@@ -1,35 +1,35 @@
 ---
-title: Gatsby.js Documentation
+शीर्षक: Gatsby.js ट्यूटोरियल
 disableTableOfContents: true
 ---
 
-import EmailCaptureForm from "../../www/src/components/email-capture-form"
+आयात EmailCaptureForm from "../../www/src/components/email-capture-form"
```
Comment (arshadkazmi42) [inline docs/docs/index.md]: `आयात` => `import`

we are not translating code related key words

File: docs/docs/index.md [hunk 2]
```diff
@@ -1,35 +1,35 @@
...
 
-Gatsby is a blazing fast modern site generator for React.
+Gatsby React के लिए एक प्रज्वलन तेज़ आधुनिक साइट जनरेटर है।
```
Comment (arshadkazmi42) [inline docs/docs/index.md]: `Gatsby` => `Gatsby,`

File: docs/docs/index.md [hunk 3]
```diff
@@ -1,35 +1,35 @@
...
 
-## Get Started
+## शुरू करना
```
Comment (arshadkazmi42) [inline docs/docs/index.md]: `शुरू करना ` => `शुरू करना`

remove extra space

File: docs/docs/index.md [hunk 4]
```diff
@@ -1,35 +1,35 @@
...
 
-## Get Started
+## शुरू करना 
 
-There are two main ways to get started with Gatsby:
+Gatsby के साथ शुरुआत करने के दो मुख्य तरीके हैं:
 
-1. [Tutorial](/tutorial/): Step-by-step instructions on how to install Gatsby and start a project: written for people without Gatsby or web development experience, though these learning resources have helped developers of all skill levels.
-2. [Quick start](/docs/quick-start): One page summary of how to install Gatsby and start a new project.
+1. [ट्यूटोरियल](/tutorial/): कैसे Gatsby को स्थापित करने और एक परियोजना शुरू करने के लिए चरण-दर-चरण निर्देश: बिना Gatsby या वेब विकास अनुभव के लोगों के लिए लिखा गया है, हालांकि इन सीखने के संसाधनों ने सभी कौशल स्तरों के डेवलपर्स की मदद की है।
```
Comment (arshadkazmi42) [inline docs/docs/index.md]:
```
कैसे Gatsby को स्थापित करने और एक परियोजना शुरू करने के लिए चरण-दर-चरण निर्देश: बिना Gatsby या वेब विकास अनुभव के लोगों के लिए लिखा गया है, हालांकि इन सीखने के संसाधनों ने सभी कौशल स्तरों के डेवलपर्स की मदद की है।
```
=>
```
Gatsby, को इनस्टॉल करने और एक प्रोजेक्ट सुरु करने के लिए स्टेप-बाई-स्टेप निर्देश: ये उन लोगो के लिए बनाया गया है जिनको Gatsby या वेब विकास का कोई अनुभव नहीं है, हालाँकि इन रिसोर्सेज ने सभी लेवल के डेवेलपर्स की मदद की है।
```

File: docs/docs/index.md [hunk 5]
```diff
@@ -1,35 +1,35 @@
...
+2. [त्वरित प्रारंभ](/docs/quick-start): Gatsby को स्थापित करने और एक नई परियोजना शुरू करने का एक पृष्ठ सारांश।
```
Comment (arshadkazmi42) [inline docs/docs/index.md]: `त्वरित प्रारंभ` => `क्विक स्टार्ट`

`त्वरित प्रारंभ` sounds a bit complex.

Comment (arshadkazmi42) [inline docs/docs/index.md]: `स्थापित` => `इनस्टॉल`

please change it for whole file, something like find and replace

Comment (arshadkazmi42) [inline docs/docs/index.md]: `नई परियोजना` => `नए प्रोजेक्ट`

change `परियोजना` also to `प्रोजेक्ट` in whole file

Comment (arshadkazmi42) [inline docs/docs/index.md]: `पृष्ठ` => `पेज`

File: docs/docs/index.md [hunk 6]
```diff
@@ -1,35 +1,35 @@
...
 
-## Go further
+## आगे बढ़ो
 
-1. [Recipes](/docs/recipes/): Find some quick answers for how to accomplish some specific, common tasks with Gatsby.
-2. Choose your own adventure and peruse the various sections of the Gatsby docs:
+1. [Recipes](/docs/recipes/): Gatsby गैट्सबी के साथ कुछ विशिष्ट, सामान्य कार्यों को पूरा करने के लिए कुछ त्वरित उत्तर खोजें.
```
Comment (arshadkazmi42) [inline docs/docs/index.md]: `Gatsby गैट्सबी` => `Gatsby`

Comment (arshadkazmi42) [inline docs/docs/index.md]: `त्वरित` => `क्विक`

Comment (arshadkazmi42) [inline docs/docs/index.md]: `खोजें.` => `खोजें।`

File: docs/docs/index.md [hunk 7]
```diff
@@ -1,35 +1,35 @@
...
+2. अपना खुद का साहसिक चुनें और Gatsby डॉक्स के विभिन्न अनुभागों का उपयोग करें.
```
Comment (arshadkazmi42) [inline docs/docs/index.md]: `साहसिक` => `एडवेंचर`

I am not very sure about this. Any thoughts? Which would be better

Comment (arshadkazmi42) [inline docs/docs/index.md]: `करें.` => `करें।`

File: docs/docs/index.md [hunk 8]
```diff
@@ -1,35 +1,35 @@
...
 
-   - [Reference Guides](/docs/guides/): Learn about the many different topics around building with Gatsby, like sourcing data, deployment, and more.
-   - [Gatsby API Reference](/docs/api-reference/): Learn more about Gatsby APIs and configuration.
-   - [Releases & Migration](/docs/releases-and-migration/): Find release notes and guides for migration between major versions.
-   - [Conceptual Guide](/docs/conceptual-guide/): Read high-level overviews of the Gatsby approach.
-   - [Gatsby Internals](/docs/gatsby-internals/): Dig into how Gatsby works behind the scenes.
-   - [Using Gatsby Professionally](/docs/using-gatsby-professionally/): Learn tips and tricks for how to explain Gatsby to others at work, so that you have more opportunities to work with Gatsby professionally.
+   - [संदर्भ मार्गदर्शिकाएँ](/docs/guides/): Gatsby के साथ निर्माण सामग्री के आसपास कई अलग-अलग विषयों के बारे में जानें, जैसे सोर्सिंग डेटा, परिनियोजन, और बहुत कुछ.
```
Comment (arshadkazmi42) [inline docs/docs/index.md]: `संदर्भ मार्गदर्शिकाएँ` => `रिफरेन्स गाइड्स`

Comment (arshadkazmi42) [inline docs/docs/index.md]: `deployment` => `डिप्लॉयमेंट`

Comment (arshadkazmi42) [inline docs/docs/index.md]: `कुछ.` => `कुछ।`

Change `.` to `।` everywhere

File: docs/docs/index.md [hunk 9]
```diff
@@ -1,35 +1,35 @@
...
+   - [Gatsby एपीआई संदर्भ](/docs/api-reference/): Gatsby API और कॉन्फ़िगरेशन के बारे में अधिक जानें।
```
Comment (arshadkazmi42) [inline docs/docs/index.md]: `संदर्भ` => `रिफरेन्स`

File: docs/docs/index.md [hunk 10]
```diff
@@ -1,35 +1,35 @@
...
+   - [विमोचन और माइग्रेशन](/docs/releases-and-migration/): प्रमुख संस्करणों के बीच प्रवास के लिए रिलीज़ नोट और गाइड खोजें।
```
Comment (arshadkazmi42) [inline docs/docs/index.md]: `विमोचन` => `रीलीज़ेज़`

Comment (arshadkazmi42) [inline docs/docs/index.md]: `संस्करणों` => `वर्शनस`

Comment (arshadkazmi42) [inline docs/docs/index.md]: `प्रवास` => `माइग्रेशन्स`

File: docs/docs/index.md [hunk 11]
```diff
@@ -1,35 +1,35 @@
...
+   - [वैचारिक मार्गदर्शक](/docs/conceptual-guide/): Gatsby दृष्टिकोण के उच्च-स्तरीय साक्षात्कार पढ़ें।
```
Comment (arshadkazmi42) [inline docs/docs/index.md]: `मार्गदर्शक` => `गाइड`

Comment (arshadkazmi42) [inline docs/docs/index.md]: `साक्षात्कार` => `ओवरव्यूज`

File: docs/docs/index.md [hunk 12]
```diff
@@ -1,35 +1,35 @@
...
+   - [Gatsby आंतरिक](/docs/gatsby-internals/): पता लगाएं कैसे Gatsby पर्दे के पीछे काम करता है
```
Comment (arshadkazmi42) [inline docs/docs/index.md]: `आंतरिक` => `इंटरनल्स`

Comment (arshadkazmi42) [inline docs/docs/index.md]: `पता लगाएं कैसे Gatsby पर्दे के पीछे काम करता है` => `पता लगाएं Gatsby पर्दे के पीछे कैसे काम करता है।`

File: docs/docs/index.md [hunk 13]
```diff
@@ -1,35 +1,35 @@
...
+   - [Gatsby व्यावसायिक रूप से उपयोग करना](/docs/using-gatsby-professionally/):काम पर दूसरों को Gatsby को कैसे समझा जाए, इसके लिए टिप्स और ट्रिक्स सीखें, ताकि आपके पास Gatsby के साथ पेशेवर रूप से काम करने के अधिक अवसर हों।
```
Comment (arshadkazmi42) [inline docs/docs/index.md]: `व्यावसायिक` => `प्रोफेश्नल्ली`

Comment (arshadkazmi42) [inline docs/docs/index.md]: `:काम पर दूसरों को Gatsby को कैसे समझा जाए, इसके लिए टिप्स और ट्रिक्स सीखें,` => `: टिप्स और ट्रिक्स सीखें जिससे आप काम पर दूसरों को Gatsby कैसे समझाया जाए,`

Comment (arshadkazmi42) [inline docs/docs/index.md]: `पेशेवर` => `प्रोफेशनल`

File: docs/docs/index.md [hunk 14]
```diff
@@ -1,35 +1,35 @@
...
 
-3. Check out the [Ecosystem](/ecosystem/) libraries for Gatsby starters and plugins, as well as external community resources.
+3. Gatsby शुरुआत और प्लगइन्स, साथ ही बाहरी सामुदायिक संसाधनों के लिए [पारिस्थितिकी तंत्र](/ecosystem) पुस्तकालयों की जाँच करें।
```
Comment (arshadkazmi42) [inline docs/docs/index.md]: `पारिस्थितिकी तंत्र` => `इकोसिस्टम`

Comment (arshadkazmi42) [inline docs/docs/index.md]: `पुस्तकालयों` => `लाइब्रेरीज`

Comment (arshadkazmi42) [inline docs/docs/index.md]: `की जाँच करें` => `को देखें`

File: docs/docs/index.md [hunk 15]
```diff
@@ -1,35 +1,35 @@
...
 
-## Start contributing
+## योगदान देना शुरू करें
 
-Visit the [Contributing](/contributing/) section to find guides on the Gatsby community, code of conduct, and how to get started contributing to Gatsby.
+Gatsby समुदाय पर मार्गदर्शिकाएँ, आचार संहिता, और कैसे Gatsby के लिए [योगदान](/contributing) शुरू करने के लिए योगदान करने के लिए योगदान अनुभाग पर जाएँ।
```
Comment (arshadkazmi42) [inline docs/docs/index.md]:
```
Gatsby समुदाय पर मार्गदर्शिकाएँ, आचार संहिता, और कैसे Gatsby के लिए [योगदान](/contributing) शुरू करने के लिए योगदान करने के लिए योगदान अनुभाग पर जाएँ।
```
=>
```
Gatsby कम्युनिटी, अचार सहिंता, और Gatsby में योगदान कैसे सुरु करें इसके लिए [कंट्रिब्यूटिंग](/contributing/) अनुभाग देखें।
```

File: docs/docs/index.md [hunk 16]
```diff
@@ -1,35 +1,35 @@
...
 
 <EmailCaptureForm signupMessage="Want to keep up with the latest tips &amp; tricks? Subscribe to our newsletter!" />
```
Comment (arshadkazmi42) [inline docs/docs/index.md]:
```
Want to keep up with the latest tips &amp; tricks? Subscribe to our newsletter!
```
=>
```
क्या आप अपने आप को लेटेस्ट टिप्स और ट्रिक्स के साथ अपडेट रखना चाहते हैं? तो हमारे नूस्लेटर सब्सक्राइब करें!
```

**Review by arshadkazmi42 — CHANGES_REQUESTED** [review]
Great work.
I have added a couple of feedback, once those are fixes, this is good to go

File: docs/docs/index.md (hunk 7 above)
Comment (mrmakadia94) [reply docs/docs/index.md]: i think एडवेंचर is easy to relate the string

**Comment by mrmakadia94** [comment]
Just updated the index.md file as per @arshadkazmi42 given changes.

File: docs/docs/index.md (hunk 8 above)
Comment (arshadkazmi42) [reply docs/docs/index.md]: You missed this one.

Actually this was supposed to be `परिनियोजन` => `डिप्लॉयमेंट`

Sorry! My bad. Can you make changes for this

File: docs/docs/index.md (hunk 14 above)
Comment (arshadkazmi42) [reply docs/docs/index.md]: > `पारिस्थितिकी तंत्र` => `इकोसिस्टम`

`इकोसिस्टम तंत्र` 
remove `तंत्र` keep only `इकोसिस्टम`

File: docs/docs/index.md (hunk 16 above)
Comment (arshadkazmi42) [reply docs/docs/index.md]: `नूस्लेटर` => `न्यूज़लेटर`

Can  you change this word, `न्यूज़लेटर` this sounds better

**Comment by arshadkazmi42** [comment]
@mrmakadia94 Awesome work with the fixes.
There is two more i have added and 1 you missed. Can you work on those 3 fixes and we will get this merged

**Comment by mrmakadia94** [comment]
Thanks @arshadkazmi42  for given updates.

**Review by arshadkazmi42 — APPROVED** [review]
Awesome work 👍 
Thank you for working on it and make changes for the feedbacks

**Comment by arshadkazmi42** [comment]
@mrmakadia94 Thank you for this.
You can claim your swag from above url 🎉

**Comment by mrmakadia94** [comment]
Thanks for @arshadkazmi42 for guiding and support and Thanks for @gatsbybot providing gift :)

**Comment by arshadkazmi42** [comment]
Keep up the awesome work you are doing 👍

**Pull request merged**